scripts/genpinctrl/stm32-pinctrl-config.yaml

```diff
@@ -299,3 +299,6 @@
 
 - name: JTAG PORT
   match: ^(SYS|DEBUG)_((JTMS-)?SWDIO|(JTCK-)?SWCLK|JTDI|JTDO(-TRACESWO|-SWO)?|(NJ)?JTRST)$
+
+- name: TSC
+  match: "^TSC_(?:G\\d+_IO\\d+|SYNC)$"
```

Comment:
Please sort alphabetically. Seems that the alphabetical sorting requires some review, but let's not making it worse.

Reply:
Do you mean to move this definition somewhere in the file where `name` is in alphabetical order? or the generated pin definitions?

Reply:
> Do you mean to move this definition somewhere in the file where name is in alphabetical order? 

yup

Reply:
Got it, thanks